### ISSUE TITLE
The app.configure() function has been removed in Express 4

### ISSUE THREAD

**Issue by xx7y7xx** [fliptoo/swagger-express#36]
I saw `app.configure()` was used in `README.md` and `example/app.js`. But in the latest version of Express (4.x), The `configure` function was removed.

https://expressjs.com/en/guide/migrating-4.html#other-changes

> The app.configure() function has been removed. Use the process.env.NODE_ENV or app.get('env') function to detect the environment and configure the app accordingly.

**Comment by TangMonk** [comment]
+1

**Comment by gusgard** [comment]
+1

**Comment by rkmax** [comment]
Yeah, please update the example setup, newcomer's mind will blow up
